### PR TITLE
SOW-24: Visualisation Component

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
## Changes:
- Added Visualisation component with story
- Added 2 CSV files for example story

- [x] add unit test

### CODE REVIEW

File: lib/timetable-visualisation/Visualisation.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      await csvToJson()
+        .fromString(data)
+        .then((jsonObj) => {
+          loadedData = {
+            ...loadedData,
+            ...jsonObj[0],
+          };
+        });
+    });
+    setTimetableData(loadedData);
```
Comment (Ro4052) [inline lib/timetable-visualisation/Visualisation.tsx:57]: I don't think we want to be mixing `await` and `.then`, we can simplify this function:

```suggestion
    const stagesData = await loadCSV(stagesFilepath);
    const events = await csvToJson().fromString(stagesData);

    const headersData = await loadCSV(headersFilepath);
    const headers = await csvToJson().fromString(headersData);

    const loadedData: DevelopmentPlan = {
      ...DEFAULT_TIMETABLE_DATA,
      timetableEvents: events,
      ...headers[0],
    };

    setTimetableData(loadedData);
```

File: lib/timetable-visualisation/Visualisation.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  endDate: new Date(),
+  timetableEvents: [],
+};
+
+type VisualisationProps = {
+  stagesFilepath: string;
+  headersFilepath: string;
+};
+
+export const Visualisation = (props: VisualisationProps) => {
```
Comment (Ro4052) [inline lib/timetable-visualisation/Visualisation.tsx:30]: Do we need to export this component from `main.ts`?

File: lib/timetable-visualisation/Visualisation.tsx (hunk 1 above)
Comment (vferraro-scottlogic) [reply lib/timetable-visualisation/Visualisation.tsx:57]: Thanks Ronan :) I lifted this part from plan-making code and didn't realise we were mixing .then()s and awaits

Comment (vferraro-scottlogic) [reply lib/timetable-visualisation/Visualisation.tsx]: change applied 👍

**Review by Ro4052 — DISMISSED** [review]
Approved pending unit tests

**Review by Ro4052 — APPROVED** [review]

**Pull request merged**